src/server/game/Entities/Player/Player.h

```diff
@@ -2250,6 +2250,17 @@ class Player : public Unit, public GridObject<Player>
         std::string GetMapAreaAndZoneString() const;
         std::string GetCoordsMapAreaAndZoneString() const;
 
+        // Cinematic camera data and remote sight functions
+        uint32 GetActiveCinematicCamera() const { return m_activeCinematicCameraId; }
+        void SetActiveCinematicCamera(uint32 cinematicCameraId = 0) { m_activeCinematicCameraId = cinematicCameraId; }
+        bool onCinematic() const { return (m_cinematicCamera != nullptr); }
```

Comment:
Shouldn't be lower camel cased imo, doesn't follow the naming convention for the _majority_ of TC's functions (or rather, your other functions in this same PR as well).